src/server/tcp.rs

```diff
@@ -439,6 +466,7 @@ mod tests {
         }
     }
 
+<<<<<<< HEAD
```

Comment:
The latest push contains unmerged changes.

Reply:
Thanks! So weird that Git and `cargo` didn't catch that...